### PR TITLE
Added a code of conduct.

### PR DESCRIPTION
**Pull request by glynos**
Like I said I'd do last month, I've added a code of conduct.

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (deanberris) [inline code_of_conduct.md]: [INSERT EMAIL ADDRESS] -- maybe we should have one set up first? Should we be using a mailing list in the cpp-netlib.org domain? I'd be happy to set that up if you think that would be useful.

**Comment by deanberris** [comment]
Thank you @glynos for doing this!

**Comment by glynos** [comment]
I just added our e-mail address, which are already in the README file.

**Comment by deanberris** [comment]
LGTM

Thanks Glyn!

**Pull request merged**